tests/conftest.py

```
@@ -0,0 +1,175 @@
+from datetime import datetime, timedelta, timezone
+from pathlib import Path
+from typing import Callable, Optional
+
+import jwt
+import pytest
+from fastapi.testclient import TestClient
+
+# Configure SQLite path for tests before importing FastAPI app
+TEST_DB_PATH = Path(__file__).resolve().parent / "test.sqlite3"
+TEST_DB_PATH.parent.mkdir(parents=True, exist_ok=True)
+# Always use an isolated SQLite file for tests, regardless of any pre‑existing env.
```

Ruff (0.14.10)

Comment contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)